README-mux.md

```diff
@@ -0,0 +1,345 @@
+
+You will need to configure sudo to not require a tty e.g. something like this:
+
+    # ls -l /etc/sudoers.d/999-cloud-init-requiretty
+    -r--r-----. 1 root root 21 Dec  1 13:17    /etc/sudoers.d/999-cloud-init-requiretty
+    # cat /etc/sudoers.d/999-cloud-init-requiretty
+    Defaults !requiretty
+
+You may need to disable SELinux `sudo setenforce Permissive` or edit
+`/etc/selinux/config` and use `SELINUX=permissive`
+
+You will need to ensure NetworkManager is running:
+
+    # systemctl daemon-reload
+    # systemctl enable NetworkManager
+    # systemctl start NetworkManager
+
+## Installing ViaQ
+
+These instructions and config files are for an all-in-one, single machine, run
+ansible on the same machine you are installing ViaQ on.
+
+*NOTE* *THIS IS NOT FOR PRODUCTION USE*
+
+The setup below is for a developer or demo machine, all-in-one, running
+Ansible in *local* mode to install ViaQ on the same machine as Ansible is
+running on.  It also configures the `AllowAllPasswordIdentityProvider` which
+means anyone can log in using the OpenShift UI.
+
+Ansible is used to install ViaQ and OCP or Origin using OpenShift Ansible.
+The following packages are required: openshift-ansible
+openshift-ansible-callback-plugins openshift-ansible-filter-plugins
+openshift-ansible-lookup-plugins openshift-ansible-playbooks
+openshift-ansible-roles
+
+    # yum install openshift-ansible \
+      openshift-ansible-callback-plugins openshift-ansible-filter-plugins \
+      openshift-ansible-lookup-plugins openshift-ansible-playbooks \
+      openshift-ansible-roles
+
+If the 3.5/1.5 versions of these packages are not available, you can use the
+git repo `https://github.com/openshift/openshift-ansible.git` and the
+`release-1.5` branch:
+
+    # git clone https://github.com/openshift/openshift-ansible.git -b release-1.5
+
+You will need to use the `ansible-playbook` command with an Ansible inventory
+file and a `vars.yaml` file.  You should not have to edit the inventory file.
+All customization can be done via the `vars.yaml` file.
+
+Download the files [vars.yaml.template](vars.yaml.template) and
+[ansible-inventory-origin-15-aio](ansible-inventory-origin-15-aio)
+
+    # curl https://raw.githubusercontent.com/ViaQ/Main/master/vars.yaml.template > vars.yaml.template
+    # curl https://raw.githubusercontent.com/ViaQ/Main/master/ansible-inventory-origin-15-aio > ansible-inventory
+
+To use ViaQ on Red Hat OCP, use the
+[ansible-inventory-ocp-35-aio](ansible-inventory-ocp-35-aio) file:
+
+    # curl https://raw.githubusercontent.com/ViaQ/Main/master/ansible-inventory-ocp-35-aio > ansible-inventory
+
+Copy `vars.yaml.template` to `vars.yaml`.  You will need to change the
+following fields in `vars.yaml`:
+
+* `ansible_ssh_user` - this is the user created in
+  [provisioning](#provisioning-a-machine-to-run-viaq) which can use
+  passwordless ssh
+* `ansible_become` - use `no` if `ansible_ssh_user` is `root`, otherwise,
+  use `yes`
```

Comment:
I see that the file has "true". So perhaps need to update to "true" and "false" here?